lib/daysOfTheWeek.js

```diff
@@ -1,4 +1,4 @@
-module.exports.getDayName = (date) => {
+module.exports.getDayName = (date = new Date()) => {
```

Comment:
The `getDayName` function has been changed.  It only changed a little bit, but this could affect the tests, so we should also go look at those.